### DISCUSSION TITLE
窗口显示不全，也没法拉伸

### DISCUSSION THREAD

**Discussion by ilixtic** [cangzhang/ChampR#263] [General]
![image](https://github.com/cangzhang/champ-r/assets/120478622/91aed5a8-47e5-4baf-93ae-b8b4176af93a)
不知道为什么会这样，窗口显示不全，然后也没法拉伸，不像1.0那样可以随意拉伸窗口，
我的电脑屏幕分辨率如下
![image](https://github.com/cangzhang/champ-r/assets/120478622/9449dc21-481a-40a6-a820-d3c50b4eff40)

**Comment by cangzhang** [comment]
试试这里 
https://www.zhihu.com/question/54913665

**Reply by ilixtic** [reply]
我设置了，但是没有用
![image](https://github.com/cangzhang/champ-r/assets/120478622/18af0995-70be-45a6-9e7a-e64863d65341)
![image](https://github.com/cangzhang/champ-r/assets/120478622/c3eeb65d-ac04-4c80-993b-21067b507c9c)

**Reply by ilixtic** [reply]
我这个是1920*1080，也不是高分辨率屏幕，而且放缩也是100%，没有改，所以我也不知道是啥情况。我看tg上也有其他老哥是这情况

**Comment by cangzhang** [comment]
久等了，试试新版本 https://github.com/cangzhang/champ-r/releases/tag/v2.0.2-b1

**Reply by ilixtic** [reply]
![image](https://github.com/cangzhang/champ-r/assets/120478622/16166f3c-9b8c-4a61-8e5e-63ea6a0655cc)
新版本好像也用不了，不知道啥情况
